Makefile

```diff
@@ -39,8 +42,16 @@ all: test build
 
 test: clean cover.out
 
-unit:
-	go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
+unit: kubebuilder
+	$(KUBEBUILDER_ENV) go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
```

Comment:
is there a way we can tie envtest tests to e2e instead of unit? there's a substantial startup cost, right?

Reply:
the tests execute in just under 4 seconds -- locally, they seem to start pretty quickly.